src/components/layout/SetupLayout.tsx

```diff
@@ -9,10 +8,8 @@ interface SetupLayoutProps {
 
 export function SetupLayout({ children }: SetupLayoutProps) {
   return (
-    <NoSSR>
-      <Web3Provider>
-        <div className={styles.landingContainer}>{children}</div>
-      </Web3Provider>
-    </NoSSR>
+    <Web3Provider>
```

Comment:
You don't need a different layout just for the setup page, you can use the Web3Provider with the other context provider components for clarity.